UI/src/app/components/comfy/comfy.component.html

```diff
@@ -5,6 +5,7 @@ <h4>Filter by source columns
   <app-comfy-search-by-name *ngIf="data.source?.length"
                             class="filter-by-name"
                             placeholder='Search by source column'
+                            criteria="{{data.sourceColumnsfilter}}"
```

Comment:
```ts
[criteria]="data.sourceColumnsfilter"
```

is better to use because [] binding in angular doesn't stringify passed object, when {{}} does

Reply:
Done